src/components/teaser-paid/teaser-paid.stories.ts

```diff
@@ -0,0 +1,130 @@
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { InputType } from '@storybook/types';
+import type { Args, ArgTypes, Decorator, Meta, StoryObj } from '@storybook/web-components';
+import { html, type TemplateResult } from 'lit';
+
+import { sbbSpread } from '../core/dom';
+import sampleImages from '../core/images';
+
+import readme from './readme.md?raw';
+
+import '../chip';
+import '../image';
+import './teaser-paid';
+
+const ariaLabel: InputType = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'link',
+  },
+};
+
+const hrefs: string[] = [
+  'https://www.sbb.ch',
+  'https://github.com/lyne-design-system/lyne-components',
+];
+const href: InputType = {
+  options: Object.keys(hrefs),
+  mapping: hrefs,
+  control: {
+    type: 'select',
+    labels: {
+      0: 'sbb.ch',
+      1: 'GitHub Lyne Components',
+    },
+  },
+  table: {
+    category: 'Link',
```

Comment:
Otherwise there are two categories called LINK
```suggestion
    category: 'link',
```